apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -443,7 +494,55 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     }
 
     if (parsedStepParam == AppOnboardingSteps.EVENT_TYPES_STEP) {
+      if (!showEventTypesStep) {
+        return {
+          redirect: {
+            permanent: false,
+            destination: `/apps/installed/${appMetadata.categories[0]}?hl=${appMetadata.slug}`,
+          },
+        };
+      }
       eventTypes = await getEventTypes(user.id, parsedTeamIdParam);
+      if (isConferencing) {
```

Comment:
We need this additional data for the Locations dropdown

Reply:
@SomayChauhan can we get the location options from another trpc endpoint? I don't think we should bloat NEXT_DATA even further. Also I don't think this is a "critical first-render needed" data.

I'm noticing this page is getting quite bloated because of this.

<img width="839" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/0b5b522a-66d8-4a55-bc3a-d909eb7b885f">

Reply:
@zomars attempted a fix in https://github.com/calcom/cal.com/pull/15193/commits/2ebaf98dcc55ccbb93712e695a4b1fecae29f576